src/dirwatcher.js

```diff
@@ -0,0 +1,33 @@
+import { spawn } from 'child_process';
+import { EventEmitter } from 'events'
+import fs from 'fs';
+import util from 'util';
+import path from 'path';
+import hound from 'hound';
+import csvjson from 'csvjson';
+import Importer from './importer';
+
+export default class DirWatcher extends EventEmitter {
+  /**
+   * Watches directory and emit 'dirwatcher:changed' event
+   * after anything was added to the path folder
+   * @param {any} path
+   * @param {any} delay – TBD
+   * @memberOf DirWathce
+   */
+  watch(dirname, delay) {
+    const watcher = hound.watch(path.join(__dirname, '../', dirname));
+    watcher.on('create', (file, stats) =>
+      (path.extname(file) === '.csv') && this.emit('dirwatcher:changed',
+        Importer.importSync(file)));
```

Comment:
Iporter should use file watcher, not vise versa. Purpose of the watcher is just to emit events, without triggering actual import logic.